include/point.h

```diff
@@ -9,19 +9,30 @@ template <unsigned Tdim>
 class Point {
  public:
   //! \brief Constructor with id and coordinates
-  Point(unsigned id, const std::array<double, Tdim>& coord);
+  Point(unsigned id, const std::array<double, Tdim>& coord,
+        const std::array<double, 6>& stress);
 
   //! Returns the id of the vertex
   unsigned id() const { return id_; }
 
   //! Returns coordinates of the vertex
   std::array<double, Tdim> coordinates() const { return coordinates_; }
 
+  //! Returns initial stress of the vertex
+  std::array<double, 6> stress() const { return stress_; }
```

Comment:
You need the point class to hold the vertex info, or is it being done in the Mesh class?